finance-api/concepts/how-card-payments-work-at-Zettle.md

```diff
@@ -0,0 +1,19 @@
+How card payments work at Zettle
+===
+A merchant's Zettle account handles different status of transactions. As shown in [Figure 1 Zettle card transaction flow](#Zettle-card-transaction-flow), after a merchant takes a card payment with Zettle, the transaction for the payment will go through the following phases in sequence:
+1. Recorded in the merchant's preliminary account:
+ 
+   Zettle checks the transaction with the acquiring bank to capture the funds.
+   
+2. Recorded in the merchant's liquid account:
+
+   Zettle checks whether the payment is ready to be paid out to the merchant.
+   
+3. Recorded as payout in the merchant's liquid account: 
+  
+   Zettle makes sure that the correct amount of money will be paid out from the merchant's Zettle account to the merchant's bank account or PayPal Wallet for PayPal users. Then the merchant will receive the deposit in the upcoming payout.
+   
+> **Note:** The amount of time for the money to be paid out varies depending on the country. Check [Zettle Help Centre](https://www.zettle.com/gb/help/articles/1084784-deposits) for country-specific information.
+
+<figure ><img id="Zettle-card-transaction-flow" src="../images/Zettle-card-transaction-flow.svg" alt="This card transaction flow shows how Zettle handles card payments." >  <figcaption><b>Figure 1. </b>Zettle card transaction flow</figcaption></figure>
+
```

Comment:
<img width="531" alt="Screenshot 2021-07-07 at 12 06 17" src="https://user-images.githubusercontent.com/18021829/124741228-c3fa3d80-df1b-11eb-9874-4f57b750d800.png">

On this page, after a merchant takes a card payment with Zettle, you have listed 3 phases a transaction will go through in sequence. But on the flowchart about there are multiple steps, which can be confusing.
Also, there is a Zettle logo also on this image, does it represent anything? I found it a hard understand the relation/